core/src/main/java/org/elasticsearch/action/search/SearchRequestBuilder.java

```diff
@@ -465,7 +357,7 @@ public SearchRequestBuilder setTrackScores(boolean trackScores) {
      * the source of the document will be returned.
      */
     public SearchRequestBuilder addFields(String... fields) {
```

Comment:
nit: I wonder if the "add" prefix is ok , seems more like a set to me, not sure if this was already the case before though